.circleci/config.yml

```diff
@@ -146,7 +92,70 @@ jobs:
       - image: << parameters.dist >>:<< parameters.release >>
     working_directory: /workspace
     steps:
-      - << parameters.dist >>_install_build_deps
+      - run:
+          name: Possibly activate centos:8 extra repos
+          command: |
+            if [ << parameters.dist >> = centos ]; then
+                if [ << parameters.release >> = 8 ]; then
+                    dnf install -y 'dnf-command(config-manager)'
+                    yum config-manager --set-enabled PowerTools
+                    yum install -y diffutils python3-sphinx
+                else
+                    yum install -y python-sphinx
+                fi
+                yum install -y epel-release
+                yum install -y \
+                    automake \
+                    jemalloc-devel \
+                    libconfig-devel \
+                    libcurl-devel \
+                    libedit-devel \
+                    libtool \
+                    libunwind-devel \
+                    make \
+                    openssh-clients \
```

Comment:
I think we do not need
* libconfig
* libcurl
* openssh-client

Reply:
indeed